### PR TITLE
Package ez_api.1.1.1

### PR DESCRIPTION
**Pull request by maxtori**
### `ez_api.1.1.1`
Easy API library and tools

---
* Homepage: https://github.com/ocamlpro/ez_api
* Source repo: git+https://github.com/ocamlpro/ez_api.git
* Bug tracker: https://github.com/ocamlpro/ez_api/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/ez_api/ez_api.1.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ocamlpro/ez_api"
+doc: "https://ocpmax.github.io/ez_api/ez_api/index.html"
+bug-reports: "https://github.com/ocamlpro/ez_api/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08"}
+  "json-data-encoding" {>= "0.7.0"}
+  "lwt" {>= "4.0.0"}
+  "ezjsonm" {>= "1.0.0"}
```
Comment (mseri) [inline packages/ez_api/ez_api.1.1.1/opam:21]:
```suggestion
  "ezjsonm" {>= "1.1.0"}
```

possibly more of the fixes added in https://github.com/ocaml/opam-repository/pull/20150 are needed

**Comment by maxtori** [comment]
I don't really understand why the Ci fails on result library, It's not supposed to have changed since 4.08

**Comment by mseri** [comment]
You need a conflict with result < 1.5 like in https://github.com/ocaml/opam-repository/pull/20150

File: packages/ez_api/ez_api.1.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "lwt_log"
+  "tls"
+  "httpaf-lwt-unix" {>= "0.6.0"}
+  "ocurl" {>= "0.8.0"}
+  "ezjs_fetch" {>= "0.2"}
+  "websocket-lwt-unix"
+  "websocket-httpaf-lwt"
+  "ppxlib" {>= "0.18.0"}
+  "ppx_deriving_encoding" {>= "0.3.0"}
+  "digestif" {>= "1.0.0"}
```
Comment (kit-ty-kate) [inline packages/ez_api/ez_api.1.1.1/opam:39]:
```suggestion
  "js_of_ocaml-lwt"
  "calendar"
  "cohttp-lwt-unix"
  "cohttp-lwt-jsoo"
  "lwt_log"
  "tls"
  "httpaf-lwt-unix"
  "ocurl"
  "ezjs_fetch"
  "websocket-lwt-unix"
  "websocket-httpaf-lwt"
  "ppxlib"
  "ppx_deriving_encoding"
  "digestif"
```

**Comment by mseri** [comment]
Thanks!

**Pull request merged**